test/unit/addons/utils/BufferGeometryUtils.tests.js

```diff
@@ -0,0 +1,60 @@
+import { BufferGeometry } from '../../../../src/core/BufferGeometry.js';
```

Comment:
I understand that there is no maintenance for testing things outside src folder. However, I think adding tests for addons when possible would contribute to their stability.

Reply:
If you want to add your unit test, you have to revert some parts of #23352.

- Create an `addons/utils` folder in `unit` and place your tests there.
- Create `test/unit/three.addons.unit.js`.
- Introduce a new script `test-unit-addons` in `package.json` which should look like so:

```
"test-unit-addons": "qunit -r failonlyreporter -f !-webonly test/unit/three.addons.unit.js",
```

Reply:
Thanks a lot for the hints.
I read the issue related to that commit and I understand the motivation.
On another hand, when I look at `ThreeJs` I see it as a fully-fledged framework with all "batteries included".
I'm asking myself, what would `ThreeJS` be without FBX/GLTF loaders, without OrbitControls and all that super useful stuff that lives in `addons`.
IMO it would be like a car made of the engine only (no wheels, not anything else) :)
This is not for me to decide the importance of "addons" but I thank you for allowing adding back the means to test `examples/jsm` stuff.